pom.xml

```diff
@@ -59,6 +59,7 @@
     <findbugs.maxRank>9</findbugs.maxRank>
     <hadoop.version>2.6.3</hadoop.version>
     <logback.version>1.1.3</logback.version>
+    <maven.test.redirectTestOutputToFile>true</maven.test.redirectTestOutputToFile>
```

Comment:
It looks like `log.error` messages go to the log.  How about we remove this?  I'll create a follow on ticket and I will remove convert any `System.out.println` to `log.debug` in tests.